docs/book/src/migration/migration_guide_v2tov3.md

```diff
@@ -0,0 +1,239 @@
+# Migration from v2 to v3
+
+Make sure you understand the [differences between Kubebuilder v2 and v3](./v2vsv3.md)
+before continuing
+
+Please ensure you have followed the [installation guide](/quick-start.md#installation)
+to install the required components.
+
+## Steps to migrate to v3 project layout
+
+<aside class="note warning">
+<h1>Note</h1>
+
+The following steps will be based on the [QuickStart][QuickStart] steps. 
+
+</aside>
+
+- Add the `projectName`
+
+The project name is the name of the project directory in lowercase:
+
+```
+...
+projectName: example
+...
+```
+
+- Add the `layout`
+
+The default plugin layout which is equivalent to the previous versions is `go.kubebuilder.io/v2`: 
+
+```
+...
+layout: go.kubebuilder.io/v2
+...
+```
+
+- Add the `version`
+
+The `version` field represents the version of Project layouts. So, you ought to update this to `3-alpha`:
+
+```
+...
+version: 3-alpha`
+...
+```
+
+The final PROJECT file would look like:
+
+```sh
+domain: my.domain
+layout: go.kubebuilder.io/v2
+projectName: example
+repo: example
+resources:
+- group: webapp
+  kind: Guestbook
+  version: v1
+version: 3-alpha
+```
+
+### Verification
+
+Finally, we can run `make` and `make docker-build` to ensure things are working
+fine.
+
+## Migrating your projects to use v3+ plugins
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+`v3+` plugins are still in alpha phase and are not scaffolded by default. You are able to use them, try it out and get all improvements made so far by initializing projects with the arg `--plugins=go.kubebuilder.io/v3-alpha`: 
+
+```sh 
+kubebuilder init --domain my.domain --plugins=go.kubebuilder.io/v3-alpha
+```
+
+</aside>
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+Currently, the plugin `v3-alpha` has NO breaking changes. However, until it is declared stable breaking changes can be made related to K8s API deprecations of `v1beta1` versions of `CustomResourceDefinition` and `ValidatingWebhookConfiguration` andd to upgrade the `cert-manager`. More info:
+
+- [cert-manager related configuration should be migrated to cert-manager.io/v1 #1666
+](https://github.com/kubernetes-sigs/kubebuilder/issues/1666)
+- [Set preserveUnknownFields to false in the CRD conversion webhook patch #933](https://github.com/kubernetes-sigs/kubebuilder/issues/933)
+- [Migrate existing KB project to v1 CRDs and Webhooks with minimal user effort #1065
+](https://github.com/kubernetes-sigs/kubebuilder/issues/1065)
+   
+</aside>
+
+### Prerequisites
+
+Following the above steps to migrate your project to
+v3 project layout.
```

Comment:
```suggestion
```

This says nothing new, we are at the middle of a v2->v3 migration guide, we already now we are following these steps to migrate.

Reply:
Oks. Make sense 👍